yt_dlp/extractor/fc2.py

```diff
@@ -22,8 +22,25 @@ class FC2IE(InfoExtractor):
         'md5': 'a6ebe8ebe0396518689d963774a54eb7',
         'info_dict': {
             'id': '20121103kUan1KHs',
-            'ext': 'flv',
             'title': 'Boxing again with Puff',
+            'ext': 'mp4',
+            'thumbnail': 're:https?://.+\\.jpe?g',
+        },
+        'file_minsize': 633,
+        'params': {
+            # Video too small to pass test
+            'skip_download': 'm3u8',
```

Comment:
`'skip_download': 'm3u8'` is the conventional way to communicate that the hls fragments are too small to download in tests
```suggestion
        'params': {
            'skip_download': 'm3u8',
```